bin/markdownlint-all

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+set -eu
+
+bindir=$( cd "${0%/*}" && pwd )
+
+"$bindir"/markdownlint ./*.md
```

Comment:
Can you move this call directly into the `static_checks.yml` action, and get rid of this file? I don't think we require this intermediary, specially since there is no `node_modules` to ignore.

Reply:
There seems to be `node_modules` even in this case, as once markdownlint is installed, those are generated.

I've moved this to `static_checks.yml` while still having `--ignore node_modules` there.